src/frontend/src/common/redux/utils/groupUtils.ts

```diff
@@ -14,7 +14,15 @@ export const ensureValidGroup = async (): Promise<void> => {
 
   const { dispatch, getState } = store;
   const state = getState();
-  const rawUserInfo = await fetchUserInfo();
+  let rawUserInfo: RawUserRequest | null = null;
+  try {
+    rawUserInfo = await fetchUserInfo();
+  } catch (e) {
+    return;
+  }
+  if (rawUserInfo == null) {
```

Comment:
Should be triple `===`, not double

Reply:
(aside: I would have thought our linter would flip out over this? I guess not. We should probably configure it that way.)